lib/editor-adapter/block-input.css

```
@@ -0,0 +1,82 @@
+/* BlockInput — thin block-mode input layer with textarea overlay */
+
+.block-editor {
+  position: relative;
+  font-family: 'Inter', system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)